src/ray/raylet/worker_pool.cc

```diff
@@ -497,13 +514,22 @@ Status WorkerPool::RegisterWorker(const std::shared_ptr<WorkerInterface> &worker
   return Status::OK();
 }
 
-void WorkerPool::OnWorkerStarted(const std::shared_ptr<WorkerInterface> &worker) {
+bool WorkerPool::OnWorkerStarted(const std::shared_ptr<WorkerInterface> &worker) {
```

Comment:
I think the meaning of return value is not intuitive. Can we change it to this?
```suggestion
void WorkerPool::OnWorkerStarted(const std::shared_ptr<WorkerInterface> &worker, bool *worker_available) {
```

Reply:
I think this way is not convenient. I have add doc to the function definition.

Reply:
Why don't we just combine this function with handleWorkerAvailable and make that return true false for worker_available?

Reply:
This function is used only once, and `handleWorkerAvailable` is always coming together

Reply:
The meaning of `OnWorkerStarted ` return value is that whether the started worker assigned to an async `PopWorker` request or not. So @ericl, does `worker_started_successfully` is a suitable name？Maybe `worker_available ` is better?

Reply:
@rkooo567 I don't understand. Do you mean call `OnWorkerStarted ` inside `handleWorkerAvailable `?

Reply:
Yeah that's what I meant (and having worker_available as a return value makes more sense in this case)